language/move-lang/tests/stdlib_sanity_check.rs

```diff
@@ -21,7 +22,18 @@ fn sanity_check_testsuite(path: &Path) -> datatest_stable::Result<()> {
     let (files, units_or_errors) = move_compile_no_report(&targets, &[], sender)?;
     let errors = match units_or_errors {
         Err(errors) => errors,
-        Ok(units) => move_lang::to_bytecode::translate::verify_units(units).1,
+        Ok(units) => {
+            batch_verify_modules(
+                units
+                    .iter()
+                    .filter_map(|compiled_unit| match compiled_unit {
+                        CompiledUnit::Module(_, m) => Some(m.clone()),
+                        CompiledUnit::Script(_, _) => None,
+                    })
+                    .collect(),
+            );
+            move_lang::to_bytecode::translate::verify_units(units).1
```

Comment:
This is close, but unfortunately the script isn't being linked against. Maybe its not an issue worth solving right now, as eventually we will want to actually run the std lib scripts